packages/mongodb-chatbot-ui/src/FloatingActionButtonTrigger.tsx

```diff
@@ -22,26 +22,16 @@ const styles = {
   `,
 };
 
-export type FloatingActionButtonTriggerProps = ChatbotTriggerProps & {
-  text?: string;
-};
-
-export function FloatingActionButtonTrigger(
-  props: FloatingActionButtonTriggerProps
-) {
-  const { darkMode } = useDarkMode(props.darkMode);
-  const { chatbotName, openChat } = useChatbotContext();
-  const { className, text } = props;
-
-  const buttonText = text ?? `Ask ${chatbotName ?? "the Chatbot"}`;
+export type FloatingActionButtonTriggerProps = ActionButtonTriggerProps;
 
+export function FloatingActionButtonTrigger({
```

Comment:
might be handy to note in a doc comment what the styles you apply to the ActionButtonTrigger are. like will the Floating action button trigger always be bottom right w/ a certain z index?